docs/sources/menu.yaml

```diff
@@ -345,6 +345,8 @@
           link: /developers/plugins/metadata/
         - name: Data frames
           link: /developers/plugins/data-frames/
+        - name: Variables
```

Comment:
Please make this side menu label more descriptive. Plugin variables? Variable support?